ako-gateway-api/status/gateway_status.go

```diff
@@ -140,31 +140,41 @@ func (o *gateway) Update(key string, option status.StatusOptions) {
 
 	status := gw.Status.DeepCopy()
 	addressType := gatewayv1.IPAddressType
-	status.Addresses = append(status.Addresses, gatewayv1.GatewayStatusAddress{
-		Type:  &addressType,
-		Value: option.Options.Vip[0],
-	})
-
-	// TODO: Add a way to propagate the error from the Rest layer to status layer.
-
+	if len(option.Options.Vip) != 0 {
+		status.Addresses = append(status.Addresses, gatewayv1.GatewayStatusAddress{
```

Comment:
This append function will add a duplicate vip everytime we will send status update to the gateway